lib/elixir/pages/Typespecs.md

```diff
@@ -148,7 +148,7 @@ Types can be parameterized by defining variables as parameters; these variables
     @callback function_name(type1, type2) :: return_type
     @macrocallback macro_name(type1, type2) :: Macro.t
 
-Callbacks are used to define the callbacks functions of behaviours (see `Behaviour`).
+Callbacks are used to define the callbacks functions of behaviours (see the "Behaviours" page in the documentation for more information on behaviours).
```

Comment:
Use a link here too. Use `make publish_docs` if you want to publish locally to see if the link works. :)